README.md

```diff
@@ -69,14 +69,14 @@ This sample uses bash scripts to deploy Edge Data Store to a remote Linux edge d
 
 1. Create an Azure IoT Hub with IoT Edge enabled, see [Microsoft Docs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal)
 1. Create an Azure Container Registry with Admin user enabled, see [Microsoft Docs](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal)
-1. Build Edge Data Store container matching the edge device processor architecture (ARM32, ARM64, or AMD64), see [OSIsoft Docs](https://osisoft.github.io/Edge-Data-Store-Docs/V1/Docker/EdgeDocker.html)
+1. Build Edge Data Store container matching the edge device processor architecture (ARM32, ARM64, or AMD64), see [OSIsoft Docs](https://osisoft.github.io/Edge-Data-Store-Docs/V1/Docker/EdgeDocker_1-0.html?q=edge%20docker)
 1. Push container image to Azure Container Registry, see [Microsoft Docs](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal#push-image-to-registry)
 
    ```bash
    docker push <acrLoginServer>/edgedatastore:v1
    ```
 
-1. Configure specified IotEdgeConfigPath file ([iotedge-config.json](iotedge-config.json) by default) with required Azure IoT Edge Device Module information, specifically the required Azure Container Registry details, see [Microsoft Docs](https://osisoft.github.io/Edge-Data-Store-Docs/V1/Configuration/EdgeSystemConfiguration.html)
+1. Configure specified IotEdgeConfigPath file ([iotedge-config.json](iotedge-config.json) by default) with required Azure IoT Edge Device Module information, specifically the required Azure Container Registry details, see [OSIsoft Docs](https://osisoft.github.io/Edge-Data-Store-Docs/V1/Configuration/EdgeSystemConfiguration_1-0.html)
```

Comment:
same question here about the gitignore. since this file is intended to hold Azure login info, should we change it to iotedge-config.placeholder.json, then add iotedge-config.json to the gitignore?

Reply:
The iotedge-config.json file is deployed as part of the sample, think of it as *part* of the sample instead of sample configuration. They do have to modify it but this file is actually deployed to a device as part of the sample, so it can't really be combined with our standard sample config. We could add it to gitignore, but the sample also suggests you might use a different file altogether, it isn't required to use the iotedge-config.json file we include. 

Other note, though: This link is totally incorrect, it was intended to be a link to Microsoft documentation but looks like I never had a valid link there. I tried to find what link I was originally referring to but I can't really find anything relevant in the Microsoft documentation, so I think the only real solution is just to remove this link altogether.